### ISSUE TITLE
Include documentation on X-Microsoft-Antispam-Mailbox-Delivery header

### ISSUE THREAD

**Issue by stukey** [MicrosoftDocs/OfficeDocs-o365seccomp#442]
Can you update this doc to provide information on the relatively new X-Microsoft-Antispam-Mailbox-Delivery header? I’m seeing this header in O365 but don’t know what all the values mean. Example:

X-Microsoft-Antispam-Mailbox-Delivery:
kl:1;ucf:0;jmr:0;ex:0;auth:0;dest:I;ENG:(750119)(520011016)(520007050)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 60b0c22e-ec20-d8dc-bbc5-cfbb9e98350e
* Version Independent ID: d9a1bac2-69ef-5fa7-23f1-417d7454c4cc
* Content: [Anti-spam message headers](https://docs.microsoft.com/en-us/office365/securitycompliance/anti-spam-message-headers#feedback)
* Content Source: [SecurityCompliance/anti-spam-message-headers.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/anti-spam-message-headers.md)
* Service: **o365-seccomp**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

**Comment by denisebmsft** [comment]
@stukey thank you for raising this question. We are looking into this, and an update will be posted here soon.

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by Beth22** [comment]
I have the same campaign going to the inbox in one account and the spam folder in the other. The only difference in the headers I can see is in this header. 

Inbox: 
abwl;0;wl:0;pcwl:0;kl:0;iwl:0;ijl:0;dwl:0;dkl:0;rwl:0;ucf:0;jmr:0;ex:0;auth:1;dest:I;ENG:(5062000261)(5061607266)(5061608174)(4900115)(4920090)(6515079)(4950130)(4990090);

Spam Folder:
abwl;0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ucf:0;jmr:0;ex:0;auth:1;dest:J;OFR:SpamFilterAuthJ;ENG:(5062000261)(5061607266)(5061608174)(4900115)(4920090)(6350004)(4950130)(4990090)(9140004);RF:JunkEmail;

Are you able to give some indication as to what OFR:SpamFilterAuthJ;ENG: and RF:JunkEmail; are and what they tell us about why the messages have gone to spam?

Thank you

**Comment by ianc911** [comment]
"We are looking into this, and an update will be posted here soon"

We must have different definitions of 'soon'. My definition would be a week or two. How about yours?

**Comment by Andy9999** [comment]
It will only take a few light years.....

**Comment by karllacroix-familiprix** [comment]
Same issue, here...
` X-Microsoft-Antispam-Mailbox-Delivery:abwl:0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ucf:0;jmr:0;ex:0;auth:1;dest:J;OFR:SpamFilterAuthJ;ENG:(5062000261)(5061607266)(5061608174)(4900115)(4920090)(6350004)(4950130)(4990090)(9140004);RF:JunkEmail;`

**Comment by TokyoScarab** [comment]
Hi @stukey and thank you for raising this issue!  I'll be looking into your question and seeing how it can be used to improve the documentation.

**Comment by patschi** [comment]
Thanks for your help, @TokyoScarab! Hopefully your "soon" is a bit faster :D 
I'm also suffering from the same issue and would really appreciate clarification on this.

**Comment by TokyoScarab** [comment]
@MSFTTracyP Would you happen to have some insight on this issue?

**Comment by MSFTTracyP** [comment]
Hi all! My apologies re. the wait. If it's taking too long for us to find bandwidth to address this issue, please investigate responses that are troublesome, and that you find isn't documented anywhere, by submitting a ticket:

https://support.microsoft.com/en-us/supportrequestform/8ad563e3-288e-2a61-8122-3ba03d6b8d75

https://docs.microsoft.com/en-us/office365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=phone

That will short-circuit this whole process and give you the answers you need.

**Comment by MSFTTracyP** [comment]
Hello everyone. I have a final answer regarding this article, and I'd like to apologize for any confusion before we set out. Parts of the header information are exposed for customers to read in documents like this one, but parts are never going to be exposed and are not destined or meant for documentation. One analogy for this is that asking after certain of the codes would be like as asking after the contents of encrypted text in headers. The codes that _are_ available are published as guidance at this time. Other codes will not be published and there are no plans to publish them.

At this point, I _strongly_ suggest that if you run into a problem and/or work-stoppage associated with an undocumented header, you go to Support _immediately_ for further investigation, and to avoid headaches. There is no other way that I know of to interpret, except through raising the issue there, for the purpose of investigation.

Apologizes for any inconveniences or confusions this causes. I appreciate your feedback and questions about this and thank you for your time and concern.

**Comment by lehuizi** [comment]
Hello, I think you have no idea how to administrate a mail system. If the recipients administrator don't provide us with the neccesary information about why the mail was marked as spam, we can't solve any issues with our systems. Providers that big like microsoft should be endeavoured to solve these kind of issues and help to improve the email delivery for their customers. The best option for all involved parties would be to publicate the meanings of those ids and don't censor the opinion of people by closing issues and deleting comments because you don't like what they said.

https://github.com/MicrosoftDocs/OfficeDocs-o365seccomp/issues/409

Please consider enabling a stable email communication with your customers.

**Comment by andryyy** [comment]
Microsoft mail customers have a bad time if they fail to receive like 50% of their valid, wanted mail.

It **looks like** MS is trying to solve its spam problem by forcing smaller companies to relay over one of the bigger providers on the market.

I have not met a company hosted on O365 (even outlook.com accounts) to _not_ complain about checking their Junk folder every other hour. I don't understand this filter system. If you need to create a whitelist record for each false-positive, you should invest your time in moving to another provider.

"Hey, we have new important customer, can you please put them on our whitelist?" - what a waste of time.

**Comment by hkusulja** [comment]
I also have same issue, with two private Microsoft accounts, both using webmail, and going to Junk folder, same as here.
Still need documentation and explanation about X-Microsoft-Antispam-Mailbox-Delivery header

**Comment by scottlan-at-msft** [comment]
We do not publicly advertise the purposes of all headers as the bad guys would then be able to use them to game the system.  As a Microsoft Office 365 administrator, the best bet is to use Admin Submissions to help determine if a policy or simple FN/FP was the problem:
https://docs.microsoft.com/en-us/office365/securitycompliance/admin-submission

If that doesn't help, Office 365 Support has tools that can help decode, determine what configuration changes might be necessary, and/or can escalate to Office 365 engineering for additional analysis.  We will use the feedback to improve the Admin Submission experience.

I do not understand the comment about 50% of valid email not being received by Microsoft customers. While I empathize that you may be having some difficulties - which I'm happy to help you work through - the statistics are simply not true.

If you're _not_ a Microsoft Office 365 customer and having issues sending to Microsoft Office 365 customers, please see https://sendersupport.olc.protection.outlook.com/pm/

**Comment by johnhpercival** [comment]
Where can Outlook.com Premium customers access support for emails that are wrongly being categorised as spam? My previous attempt to engage with them on this issue was a bit like banging my head against a brick wall.

**Comment by M034B5** [comment]
> Hello everyone. I have a final answer regarding this article, and I'd like to apologize for any confusion before we set out. Parts of the header information are exposed for customers to read in documents like this one, but parts are never going to be exposed and are not destined or meant for documentation. One analogy for this is that asking after certain of the codes would be like as asking after the contents of encrypted text in headers. The codes that _are_ available are published as guidance at this time. Other codes will not be published and there are no plans to publish them.
> 
> At this point, I _strongly_ suggest that if you run into a problem and/or work-stoppage associated with an undocumented header, you go to Support _immediately_ for further investigation, and to avoid headaches. There is no other way that I know of to interpret, except through raising the issue there, for the purpose of investigation.
> 
> Apologizes for any inconveniences or confusions this causes. I appreciate your feedback and questions about this and thank you for your time and concern.

But what if "support" ignores the issue as well?
Whom should we talk / refer to in such case?

I have personally reported this issue a year and half ago and we were on a call with microsoft support, we went back and forth in calls and they redirected us to another departments yet they weren't even able to identify or tell us the reason behind this problem (as in emails being marked as spam when using a selfhosted mailing server).

I had 2 or 3 ticket numbers back then but since no one cared i scraped everything and I'm just getting used to the fact that all of my company emails are to be marked as spam on outlook's end. Which is so ridiculous tbh because other mail providers like zoho or even google works just fine.

All we're saying is that a little help and attention from your end would be nice.
Although you replied with a "final answer" yet there weren't any acts upon this.

No one is asking to publish confidential stuff but maybe collaborate a little bit on how to resolve the issue and not ignore it or close it

**Comment by hkusulja** [comment]
@scottlan-at-msft It is hard to go to get some decent level of technical support from Microsoft, mostly they are focused on publicly available documentation, including this related document.
But if Microsoft officially says for this particular public email headers "we do not want to provide public documentation", then simply ..... wou ....

I also could not get any info / concrete response for this particular header (X-Microsoft-Antispam-Mailbox-Delivery) and explanation in meaning for further troubleshooting etc.

**Comment by scottlan-at-msft** [comment]
@hkusulja Are you also looking for Outlook.com consumer support or are your questions about Office 365?  If the latter, I can't stress enough opening a support ticket in the Admin portal after submitting the samples to us (not through Outlook user support).  If you have a current or very recent Office 365 support ticket with current samples (within 10 days) you want me to look at, please send me the ticket number (via email is fine).  For consumer Outlook.com, there is a different escalation path... I'm trying to get details about that - you might have to use the in-app support - I still should be able to look up the tickets.

I can understand how frustrating a poor support experience can be - and we don't want that for anyone - but without details, it is hard to follow up. I'm not clear that this forum is the best place to resolve these issues.  The two particular headers requested are not necessarily going to help customers resolve any issues on their own - if those were the only headers I had access to, I wouldn't be able to solve any issues.

**Comment by hkusulja** [comment]
> @hkusulja Are you also looking for Outlook.com consumer support or are your questions about Office 365? If the latter, I can't stress enough opening a support ticket in the Admin portal after submitting the samples to us (not through Outlook user support). If you have a current or very recent Office 365 support ticket with current samples (within 10 days) you want me to look at, please send me the ticket number (via email is fine). For consumer Outlook.com, there is a different escalation path... I'm trying to get details about that - you might have to use the in-app support - I still should be able to look up the tickets.

In recent about 30 days I do have only outlook.com Premium support request, not the Office 365 organizational...

Also, Can you specify in this docs. document that this requested header exists by Microsoft, but it is not detailed documented due to missuse and can be given more info only via support?

**Comment by johnhpercival** [comment]
> @hkusulja For consumer Outlook.com, there is a different escalation path... I'm trying to get details about that - you might have to use the in-app support - I still should be able to look up the tickets.

My ticket number is 528916, on the email address support@olcsupport.mail.helpshift.com

Any help much appreciated...

**Comment by markdascher** [comment]
Speaking for myself, I think I zeroed in on this header because it seemed to have "secret" information that the other headers lacked. If there were a header that provided some basic, actionable information for admins, then we wouldn't need this one. Basically we just need to have something to distinguish between:
* You can fix this yourself by fixing XYZ.
* The recipient can fix this by changing XYZ.
* This was flagged due to a top-secret reason. You have to contact Microsoft to get the answer.

Right now, it feels like everything is in that third bucket. (Provided that we're not bumping into one of the obvious issues listed in https://sendersupport.olc.protection.outlook.com/pm/ and similar articles.)

**Comment by ianc911** [comment]
If you don't want to publish your secret codes, then you still need to provide better documentation and troubleshooting guidance for admins. The way it is now, the admin will look at the spam headers and not have the slightest clue as to why mail is getting flagged. Simply saying 'contact support to resolve these issues' is in no way any kind of acceptable answer. It's bad for us and it 's bad for you. Step up and provide workable documentation\tools to enable the administrator to troubleshoot this without 'contacting support'.  How about an analyzer tool in which one can paste a message, which runs it through a scanning engine and suggests possible remedial action the admin can take to alleviate the problem?

**Comment by scottlan-at-msft** [comment]
@ianc911 We have a tool - actually several tools, but the most current one is linked above:
https://docs.microsoft.com/en-us/office365/securitycompliance/admin-submission

This link is for Office 365 admins, as is the article.  While we appreciate the feedback, Outlook.com is not exactly the same as Office 365, and is out of scope for the article in question.  I will take the feedback to the Outlook.com team to see how we can improve that content.

There are multiple types of headers in Office 365:
- Ones which contain actionable things that the user/admin can do.  These are (to our knowledge) all documented in this article.  We aren't hiding anything.  If during the course of a support incident we determine that there's a new helpful header or value, we'll add it as we have always done.
- Ones which contain a history of what happened, but aren't necessarily actionable to the admin or user - only to our spam analysts or support teams.

**Comment by ianc911** [comment]
Thanks for the info. Was not aware of the tool. Will give it a whirl the next time mail is sent down in flames... :)

**Comment by lehuizi** [comment]
Hello @scottlan-at-msft,

I think the provided tools can be a problem, because we mostly interact with customers that have no clue about anything to do with mails. We just can ask them to forward the mail as file, so the headers won't get touched.

Additionally, can you provide us any useful documentation, url to a contact form, a email addres or something similar for both outlook and o365 to get some help with troubleshooting when you argument with secret informations?

---

Let's  go over a brief example:

```
X-IncomingHeaderCount: 15
X-MS-Exchange-Organization-ExpirationStartTime: 11 Sep 2019 08:39:54.8091
(UTC)
X-MS-Exchange-Organization-ExpirationStartTimeReason: OriginalSubmit
X-MS-Exchange-Organization-ExpirationInterval: 1:00:00:00.0000000
X-MS-Exchange-Organization-ExpirationIntervalReason: OriginalSubmit
X-MS-Exchange-Organization-Network-Message-Id:
da1f50b7-4d5d-4c0e-21c6-08d736939ec6
X-EOPAttributedMessage: 0
X-EOPTenantAttributedMessage: 84df9e7f-e9f6-40af-b435-aaaaaaaaaaaa:0
X-MS-Exchange-Organization-MessageDirectionality: Incoming
X-Forefront-Antispam-Report: EFV:NLI;
X-MS-Exchange-Organization-AuthSource:
SG2APC01FT115.eop-APC01.prod.protection.outlook.com
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-PublicTrafficType: Email
X-MS-UserLastLogonTime: 9/11/2019 8:39:49 AM
X-MS-Office365-Filtering-Correlation-Id: da1f50b7-4d5d-4c0e-21c6-08d736939ec6
X-Microsoft-Antispam:
BCL:0;PCL:0;RULEID:(2390118)(5000188)(711020)(4605104)(610169)(650170)(651021)(1124261)(8291501072);SRVR:SG2APC01HT036;
X-MS-TrafficTypeDiagnostic: SG2APC01HT036:
X-MS-Exchange-EOPDirect: true
X-Sender-IP: ..
X-SID-PRA: ..
X-SID-Result: PASS
X-MS-Exchange-Organization-PCL: 2
X-MS-Exchange-AtpMessageProperties: SA|SL
X-OriginatorOrg: outlook.com
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 11 Sep 2019 08:39:54.4074
(UTC)
X-MS-Exchange-CrossTenant-Network-Message-Id: da1f50b7-4d5d-4c0e-21c6-08d736939ec6
X-MS-Exchange-CrossTenant-Id: 84df9e7f-e9f6-40af-b435-aaaaaaaaaaaa
X-MS-Exchange-CrossTenant-FromEntityHeader: Internet
X-MS-Exchange-Transport-CrossTenantHeadersStamped: SG2APC01HT036
X-MS-Exchange-Transport-EndToEndLatency: 00:00:02.9317888
X-MS-Exchange-Processed-By-BccFoldering: 15.20.2241.000
X-Microsoft-Antispam-Mailbox-Delivery:
abwl:0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ucf:0;jmr:0;ex:0;auth:1;dest:J;OFR:SpamFilterAuthJ;ENG:(5062000261)(5061607266)(5061608174)(4900115)(4920090)(6350004)(4950130)(4990090)(9140004);RF:JunkEmail;
X-Message-Info:
qoGN4b5S4yoYp1N8JxzRIB1rrmds10DNELaHrWH5NJlHCVQGZ/bEntOs8xmkfaZyliQAJkwHr8/yWTfNsJjWn1zXwFSj/D4cp/xeXBVDSqX2z/HDaSxpVMpysvqIpkFZaLUyFbcQCeU10md+TCjLdPmQxRZYWxyAUH80sHfQFajFh9UMm97/Reh9hqAkWx2RgrjARv+RQuzgJmnZSB7I7Q==
X-Message-Delivery: Vj0xLjE7dXM9MDtsPTA7YT0wO0Q9MjtHRD0yO1NDTD02
X-Microsoft-Antispam-Message-Info:
NDRfOTT0twYOhR3iU5i95ccOKwHjsSaD3tMksDs7Pvm1gKykovbN+B36QCp5HcL3PGdt5RTvM2VBsCcfp3sv+YmLa5Z91ai+IHl6SkWHBInsz8SCocmAotYV33frUcigtc13XTW5KE86h+7ZtpH7mVrH2cWuokn390sgQTbE04iz2MAoUDhUhTIdYpcMWri7a+aiIeYqCLP6izhLEy8AxUOwyFpGjtzyFnoZ/4VIegqbGKtlWmhl3Ho8heEcfabfRt+D+030dPXJIKeglrRaTPKOrSXXX7DscJpCphYpz/rsgWpTJeFkOkKW1sUcNkDgLn5T6e/HdwiNpcv/3VF/jPHN65jp+qVB+sSNDs+DZA35+VfKhNGopMUFNZ70vAq/Bh0/zWBswpIN2+nwq+mwVjN8N/99M50zfk2BfMBDWUv3LNhhHjv2dy49BJlOqAYWLL0oQiF3iQjuXRWQSsClMnm+wddGVLWwhNEDy6sB3z2I8D06vFNHOnyVzr0wtzUMux0Vxd04aMVr5kNy48y6Ef5I8FSgmfJ/UCRDLUGzvskiHw9Z4wthOPQsVuPLLMLYIpfwdO3icWeBxmj/Eu26h3SVz2R4UTP5N/pwlxNg/UQbLsyU2wq1vGzRrvBW1Z9+iTNxrQlMMDrCqidArr1LtIak0qHyGQylcWutBO4p0VDsQluIXQcmEWNbRo+HEM3zR+5AWYOjT3AxwnOaehnJSOOA3OJ0VHpEUsqlTB513SHBuKBhJYwiZ0GgXTbO3+leD/tTUzDLly1oMv8OBbzOdeBEwwQL0u2J+0HB1Q4ilXJebnsl823/vhxf+fLiIHsv09mquzkYP1KHwywHhzVwudjBAIj7Lv9128x/i+7ynIMHndeMMC8+Fj4Q86S3nh70BrU7W8/ZmnDVY+wp+c63TPbtXmkqIA5jaUurYdDHnvg=
```

Can you tell me how to debug this mail header?

- **X-Forefront-Antispam-Report**: Can't get any information about the "EFV"
- **X-Microsoft-Antispam**: Definitly no information about the meanings of these IDs
- **X-MS-Exchange-AtpMessageProperties**: I can't figure out what that means because I found no documentation after a 5 minute research
- **X-Microsoft-Antispam-Mailbox-Delivery**: Same thing as for X-Microsoft-Antispam

Maybe I'm missing something. Can you tell me how you would debug this email without further assistance from Microsoft (which we apparently don't get when sending to outlook.com customers)?

**Comment by johnhpercival** [comment]
> While we appreciate the feedback, Outlook.com is not exactly the same as Office 365, and is out of scope for the article in question. I will take the feedback to the Outlook.com team to see how we can improve that content.

@scottlan-at-msft, I'm really grateful for you pursuing this with the outlook.com team. It does seem like the same basic mechanism is underlying the headers, which is why I've ended up on this thread trying to get answers. I supplied my ticket reference above, so if you're able to get any further with this that would be amazing.
